pkg/dispatcher/grpc/fntypes/fntypes.proto

```diff
@@ -14,21 +14,14 @@
  * limitations under the License.
  */
 syntax = "proto3";
-import "google/protobuf/empty.proto";
 
 package fntypes;
 
-message Request {
-    string body = 1;
-}
-
-/*
- * TODO: Reply can be Empty for sink behavior. Maybe we should create a separate RPC for sink.
- * For now the Dispatcher can check the return type and decide to output the Reply.
- */
-message Reply {
-    oneof type {
-	    string body = 1;
-	    google.protobuf.Empty void = 2;
+message Message {
+	message HeaderValue {
+		repeated string values = 1;
 	}
+
+    bytes payload = 1;
+	map<string, HeaderValue> headers = 2;
```

Comment:
Should we delete this from source code control since the build downloads it from function-proto?

Reply:
@ericbottard ^^

Reply:
We could, although it's not exactly the approach we have we the whole `vendor/` directory for example. The makefile would need tweaking, as it currently doesn't trigger the download